Add Japanese Translation to GDLauncher-CarbonAdd

kazaminosuke opened the pull request:
I want to add Japanese translations to GDLauncher-Carbon.
I haven't been able to set up a test environment, so I'm not sure if it's correct.

kazaminosuke committed:
Create a

kazaminosuke committed:
Add files via upload

kazaminosuke committed:
Delete packages/i18n/locale/japanese/a

blarfoon committed:
Merge branch 'develop' into develop

MrT-tak commented:
@blarfoon huh, you should close the other PR since this is a duplicate then.

MrT-tak commented:
@blarfoon I prefer this one over #445 since this one makes sense and did not need my commits 
(I haven't still figured out how to commit to PRs properly 😭)

This does a better job then whatever I would have been able to do

kazaminosuke committed:
Latest update最新に対応

The latest minor fixes have been implemented.
最新にしまた細かい修正を行った

kazaminosuke committed:
機械翻訳によるおかしい部分を修正/Correcting the awkward parts produced by machine translation

機械翻訳によって誤った翻訳になっているところを一部修正しました/Some parts that were incorrectly translated by machine translation have been corrected.

shishen575 commented:
Should I close mine?

kazaminosuke commented:
@shishen575 
別に閉めなくていいですよあなたが閉めたほうがいいと思ったら閉めてください/You don't have to close it if you don't want to. If you think it's better to close it, then go ahead and close it.

MrT-tak commented:
どうかな？ 
僕はこっちのほうがもっと分かりやすい気がします

kazaminosuke commented:
@blarfoon 
Review required
At least 1 approving review is required by reviewers with write access.

kazaminosuke committed:
Merge branch 'develop' into develop

kazaminosuke commented:
言語ファイルが大幅に変更されたので作り直します

blarfoon commented:
Hi, unfortunately I had to make many changes to the translation system to be able to accommodate for many languages and to make it easier to maintain them.

I agree opening a new PR is probably the easiest way.

You can find everything about the new translation system here https://github.com/gorilla-devs/GDLauncher-Carbon/blob/develop/CONTRIBUTE_LANGS.md.

I noticed there is also another open PR for Japanese, since translations are now split into many files you could work together and split the work between you two.

https://github.com/gorilla-devs/GDLauncher-Carbon/pull/445

@shishen575  @MrT-tak for visibility
